tests/i18n/tests.py

```diff
@@ -86,6 +86,11 @@ def test_plural(self):
         self.assertEqual(ngettext("%(size)d byte", "%(size)d bytes", 0) % {'size': 0}, "0 octet")
         self.assertEqual(ngettext("%(size)d byte", "%(size)d bytes", 2) % {'size': 2}, "2 octets")
 
+    def test_plural_null(self):
+        self.assertEqual(trans_null.ngettext("%d year", "%d years", 0) % 0, "0 year")
```

Comment:
In English on 1 is singular. That should be `0 years`.

Reply:
yes, you right. it seems every thins except +1 and -1 are plural in English.